docs/testing/child.md

```diff
@@ -0,0 +1,3 @@
+<variable name = "title">Title</variable>
+<variable name = "child_src">testimportvariables.md</variable>
+<variable name = "child_src_working">testimportvariables</variable>
```

Comment:
Remove the spaces around the `=`.

Reply:
Btw where's the HTML output for all these test cases? :P

Reply:
I added text saying there shouldn't be any variables shown.